client/my-sites/plan-features/index.jsx

```diff
@@ -90,6 +92,15 @@ export default connect( ( state, ownProps ) => {
 		rawPrice: getPlanRawPrice( state, planProductId /**, get from abtest **/ ),
 		planConstantObj: plansList[ ownProps.plan ],
 		billingTimeFrame: get( planObject, 'bill_period_label', '' ),
+		onUpgradeClick: ownProps.plan === PLAN_FREE ? noop : () => {
```

Comment:
If we end up reusing this function in another area, we should consider moving it to a util file. I'd imagine we'd eventually convert the cart to Redux, so we can trigger a standard Redux action in the future.